spec/Snt/Capi/Repository/Article/FindBySectionParametersSpec.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace spec\Snt\Capi\Repository\Article;
+
+use InvalidArgumentException;
+use PhpSpec\ObjectBehavior;
+use Snt\Capi\Http\ApiHttpPathAndQuery;
+use Snt\Capi\Repository\Article\FindBySectionParameters;
+use Snt\Capi\Repository\FindParametersInterface;
+
+/**
+ * @mixin FindBySectionParameters
+ */
+class FindBySectionParametersSpec extends ObjectBehavior
+{
+    const URL_PATTERN = 'publication/%s/sections/%s/latest?%s';
+
+    const PUBLICATION_ID = 'sa';
+
+    const SECTION_NAME = 'bolig';
+
+    function let()
+    {
+        $this->beConstructedThrough('createForPublicationIdAndSections', [self::PUBLICATION_ID, [self::SECTION_NAME]]);
+    }
+
+    function it_can_be_constructed_through_named_constructor()
+    {
+        $this->shouldHaveType(FindBySectionParameters::class);
+        $this->shouldHaveType(FindParametersInterface::class);
+    }
+
+    function it_builds_url()
+    {
+        $path = sprintf(self::URL_PATTERN, self::PUBLICATION_ID, self::SECTION_NAME, '');
```

Comment:
to be honest, i would create constant for last parameter and use it as `self::EMPTY_QUERY` (or similar). with that i don't need to scroll up and see how `URL_PATTERN` looks like , to understand what `''` means. what do you think?